CONTRIBUTING.md

```diff
@@ -20,7 +20,11 @@ We enforce code formatting with [`gufumpt`](https://github.com/mvdan/gofumpt) (a
 "editor.formatOnSave": true,
 "gopls": {
     "formatting.gofumpt": true
-}
+},
+"go.lintTool": "golangci-lint",
+"go.lintFlags": [
+    "--fast"
```

Comment:
if this is given, does it really run all given linters? try with `-v` option to see

Reply:
yes.
https://golangci-lint.run/usage/integrations/#editor-integration

Reply:
they recommend this, otherwise it can freeze your editor

Reply:
👀 really? I can't find the typo

Reply:
gufumpt

Reply:
✅